muxdb/engine/leveldb.go

```diff
@@ -112,14 +114,20 @@ func (ldb *levelEngine) Bulk() kv.Bulk {
 		return batch
 	}
 	flush := func(minSize int) error {
-		if batch != nil && len(batch.Dump()) >= minSize {
-			if batch.Len() > 0 {
-				if err := ldb.db.Write(batch, &writeOpt); err != nil {
-					return err
+		if batch != nil {
+			if batchBytesNo := len(batch.Dump()); batchBytesNo >= minSize {
+				if batch.Len() > 0 {
+					startTime := mclock.Now()
+					if err := ldb.db.Write(batch, &writeOpt); err != nil {
+						return err
+					}
+					batchWriteElapsed := mclock.Now() - startTime
+					metricBatchWriteBytes().Set(int64(batchBytesNo))
+					metricBatchWriteDuration().Observe(time.Duration(batchWriteElapsed).Milliseconds())
```

Comment:
Is the mclock is required ?
I think the `time` package, `time.Now()` and `time.Since()` should work well.

Reply:
replace by `time`, thanks.